src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -122,14 +123,26 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
     );
 
     const navigateToCategorySettings = (category: PolicyOption) => {
+        if (backTo) {
+            Navigation.navigate(ROUTES.SETTINGS_CATEGORY_SETTINGS.getRoute(policyId, category.keyForList, backTo));
+            return;
+        }
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(policyId, category.keyForList));
```

Comment:
NAB - maybe in a follow-up we can clean this up by passing `backTo` always? so we don't have so many "if backTo exists, pass it, otherwise don't" checks? I also think it looks like `ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute` already handles backTo existing or not, right?

Reply:
Oh geez i missed that one is SETTINGS_CATEGORY_SETTINGS, one is WORKSPACE_CATEGORY_SETTINGS. Please forgive me 😅